packages/snap-preact/components/src/components/Molecules/Result/Result.test.tsx

```diff
@@ -92,6 +92,7 @@ describe('Result Component', () => {
 
 	it('should display a fallback image', () => {
 		mockResults[1].mappings!.core!.imageUrl = '';
+		(mockResults[1] as Product).images = undefined;
```

Comment:
We should revisit this, as I'm pretty sure we don't want to use `images` here any more, instead we will favor the use of the mask.

Reply:
(need to remove it from the store)